cmd/juju/application/flags.go

```diff
@@ -136,3 +139,70 @@ func (m stringMap) String() string {
 	}
 	return strings.Join(pairs, ";")
 }
+
+// configFlag records k=v attributes from command arguments
```

Comment:
most of this is the same as common.ConfigFlag - why not just modify that a bit? AFAICS, all you need is to add methods for accessing the k=v attrs and the filenames

Reply:
I didn't want to break encapsulation. Either way works for me. I changed to use common.ConfigFlag